model/time.go

```diff
@@ -163,9 +163,20 @@ func (t *Time) UnmarshalJSON(b []byte) error {
 // This type should not propagate beyond the scope of input/output processing.
 type Duration time.Duration
 
+// Set implements pflag/flag.Value
+func (d *Duration) Set(s string) (err error) {
```

Comment:
Only if they add to the documentation. `err error` is stutter and clutters the documentation. `(max int, min int)` is good and preferred over `(int, int)`.

https://go-review.googlesource.com/c/20024/